Cesium.Runtime/StdIoFunctions.cs

```diff
@@ -668,15 +667,66 @@ public static int Remove(byte* pathname)
         return 0;
     }
 
-    private static unsafe void* GetLastStream()
+    internal static StreamHandle? GetStream(void* filePtr)
     {
-        return (void*)(IntPtr)(Handles.Count - 1);
+        var handle = (IntPtr)filePtr;
+
+        var handleValue = handle.ToInt64();
+        if (handleValue is StdIn or StdOut or StdErr)
+        {
+            return Handles[(int)handleValue];
+        }
+
+        try
+        {
+            lock (_locker)
+            {
+                var gchAddr = Marshal.ReadIntPtr(handle);
+                var gch = GCHandle.FromIntPtr(gchAddr);
+
+                return gch.Target as StreamHandle;
+            }
+        }
+        catch (InvalidOperationException)
+        {
+            return null;
+        }
```

Comment:
When does this get thrown? Looks suspicious. Maybe we shouldn't handle any exception here?

Reply:
Usually only AV occurs there. In this case, the error can occur due to an invalid pointer argument when converting to GCHandle, eventually the Target property will throw an AV if converting fail.